test_projects/ios/EarlGreyExample/EarlGreyExample.xcodeproj/project.pbxproj

```diff
@@ -283,18 +283,18 @@
 				TargetAttributes = {
 					2CB7314C1C29E54A00CF35C1 = {
 						CreatedOnToolsVersion = 7.2;
-						DevelopmentTeam = H86XEK7ZTM;
+						DevelopmentTeam = AD2V26JBWL;
```

Comment:
Don't worry about that. We will change it again, once we obtain new Apple developer account